runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -79,6 +79,7 @@
     public final TransactionBuilder builder;
 
     /**
+     * TODO remove this!
```

Comment:
we have metering for that. this is completely isolated mechanism, mostly unknown and hiding. No reason to have it at this level.